pmm-app/src/pmm-settings/components/SignUp/SignUp.validators.ts

```diff
@@ -0,0 +1,20 @@
+import { Messages } from './SignUp.messages';
+
+export const required = (value: string | boolean | undefined) => (
+  value ? undefined : Messages.errors.requiredField
```

Comment:
A boolean false would give a false positive. Consider `value != null` (mind the single equal sign).

Reply:
In this case `false ? undefined : Messages.errors.requiredField` I want it to return the error message actually, since the value should be `true` in order to make it pass (think about the checkbox that must be checked). Even splitting that to two different validators, like `requireTrue` and `required` or something like that, the logic for both would be the same. Well, actually it could be more specific for the hypothetical `requireTrue` validator, the condition could be stricter: `value === true`, but I'm not sure it's worth the effort. What do you think?

Reply:
I think in case of a boolean value we expect the input to validate in either case: true or false. Null or undefined only should trigger the validation error. And this is not the same as saying `(value: boolean) => value ? undefined : Message;`

Reply:
Got your point, using a `requiredTrue` validator for that now